anta/tests/routing/bgp.py

```diff
@@ -650,10 +726,13 @@ def test(self) -> None:
 
         # Iterating over command output for different peers
         for command in self.instance_commands:
-            peer = str(command.params["peer"])
-            vrf = command.params["vrf"]
-            advertised_routes = command.params["advertised_routes"]
-            received_routes = command.params["received_routes"]
+            peer = command.params.peer
+            vrf = command.params.vrf
+            for input_entry in self.inputs.bgp_peers:
+                if str(input_entry.peer_address) == peer and input_entry.vrf == vrf:
+                    advertised_routes = input_entry.advertised_routes
+                    received_routes = input_entry.received_routes
+                    break
```

Comment:
Not sure if we should have a different way of getting the inputs (related to the other comment)